tests/gtest/avif_fuzztest_helpers.cc

```diff
@@ -193,6 +194,23 @@ std::vector<uint8_t> GetWhiteSinglePixelAvif() {
       0xc4, 0x0,  0x0,  0xaf, 0x70};
 }
 
+//------------------------------------------------------------------------------
+// Environment setup
+
+namespace {
+class Environment : public ::testing::Environment {
+ public:
+  Environment(const char* key, const char* value) : key_(key), value_(value) {}
+  void SetUp() override { setenv(key_, value_, 1); }
+  const char* key_;
```

Comment:
Nit: the data members should be private.

(Also, "name" sounds better than "key" when talking about environment variables.)

Reply:
Done inhttps://github.com/AOMediaCodec/libavif/pull/1711.